recipes.js

```diff
@@ -1,11 +1,46 @@
 const mongoose = require('mongoose');
 const Schema   = mongoose.Schema;
-const data = require('./data.js')
+const data = require('./data.js');
 
 mongoose.connect('mongodb://localhost/recipeApp')
   .then(() => {
-    console.log('Connected to Mongo!')
+    console.log('Connected to Mongo!');
   }).catch(err => {
-    console.error('Error connecting to mongo', err)
+    console.error('Error connecting to mongo', err);
   });
 
+
+  const recipeSchema = new Schema({
+    title: String,
+    level: {type: String, enum: ["Easy Peasy", "Amateur Chef", "UltraPro Chef"]},
+    ingredients: [String],
+    cousine: {type: String, required: true},
+    dishType: {type:String, enum: ["Breakfast","Dish","Snack","Drink","Dessert","Other"]},
+    image: {type: String, default:"https://images.media-allrecipes.com/images/75131.jpg"},
+    duration: {type: Number, min: [0, 'min value']},
+    creator: String,
+    created: {type: Date, default: Date.now}
```

Comment:
To avoid having an additional field in DB that' manually updated, you can use timestamps for createdAt and updatedAt:
```
const someModelSchema = new Schema({
   // fields go here
  },
  {
    timestamps:true
  }
);
```
Timestamps will create these two fields in DB.